packages/stream_chat_flutter/lib/src/thread_header.dart

```diff
@@ -61,30 +61,30 @@ class ThreadHeader extends StatelessWidget implements PreferredSizeWidget {
 
   /// Callback to call when pressing the back button.
   /// By default it calls [Navigator.pop]
-  final VoidCallback onBackPressed;
+  final VoidCallback? onBackPressed;
 
   /// Callback to call when the title is tapped.
-  final VoidCallback onTitleTap;
+  final VoidCallback? onTitleTap;
 
   /// The message parent of this thread
   final Message parent;
 
   /// Title widget
-  final Widget title;
+  final Widget? title;
 
   /// Subtitle widget
-  final Widget subtitle;
+  final Widget? subtitle;
 
   /// Leading widget
-  final Widget leading;
+  final Widget? leading;
 
   /// AppBar actions
-  final List<Widget> actions;
+  final List<Widget>? actions;
```

Comment:
Probably can make this non nullable and a const [] by default

Reply:
I wouldn't do this, if you look at the AppBar widget they don't do that